rosidl_runtime_cpp/CMakeLists.txt

```diff
@@ -41,7 +44,7 @@ if(BUILD_TESTING)
 
   ament_add_gtest(test_bounded_vector test/test_bounded_vector.cpp)
   if(TARGET test_bounded_vector)
-    target_include_directories(test_bounded_vector PUBLIC include)
+    target_link_libraries(test_bounded_vector ${PROJECT_NAME})
```

Comment:
Not that I find this incorrect but caught my attention. The test did not link to the library before? Did it work?